### DISCUSSION TITLE
Ajax call to form properties endpoint gives 404

### DISCUSSION THREAD

**Discussion by paulgoodfield** [solspace/craft-freeform#937] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**

I'm following the Vue.js integration guide - one of the first things it does in [the example](https://codesandbox.io/p/github/solspace/craft-freeform-demo-vuejs-ajax/main?file=%2Fsrc%2FQuoteForm.vue%3A44%2C5) (line 44 in `src/QuoteForm.vue`) is use `fetch` to return the relevant form properties.

When I try that in the Vue app I'm building it gives me a 404 response. The url it's hitting is `https://mylocaldomain.test/craft/freeform/form/properties/4`

The id I'm passing (4) looks to be correct - when I go into the Craft control panel and browse to the form in question, the ID number in the url is the same.

**Steps to reproduce**

1. In JS, try to hit `/craft/freeform/form/properties/{formId}` endpoint using `fetch`
2. Get 404 response

**Expected behavior**

That endpoint should return the relevant data rather than 404ing

**Craft & Plugin Info (please complete the following information):**          

- Craft Version: Craft Pro 4.5.6.1
- Freeform Version: 4.1.7
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh Install

**Comment by kjmartens** [comment, marked as answer]
Hi @paulgoodfield,

Do you have a route setup in `routes.php`? 🙂 

```
<?php
/**
 * Site URL Rules
 *
 * You can define custom site URL rules here, which Craft will check in addition
 * to any routes you've defined in Settings → Routes.
 *
 * See http://www.yiiframework.com/doc-2.0/guide-runtime-routing.html for more
 * info about URL rules.
 *
 * In addition to Yii's supported syntaxes, Craft supports a shortcut syntax for
 * defining template routes:
 *
 *     'blog/archive/<year:\d{4}>' => ['template' => 'blog/_archive'],
 *
 * That example would match URIs such as `/blog/archive/2012`, and pass the
 * request along to the `blog/_archive` template, providing it a `year` variable
 * set to the value `2012`.
 */

return [
    'graphql/api' => 'graphql/api',
    'freeform/form' => 'freeform-module/form/index',
    'freeform/form/properties/<formId:\d+>' => 'freeform-module/form/properties',
];
```
